tools/add_fib.py

```diff
@@ -29,6 +30,24 @@ def ranges(i):
 
 
 def add_fib_at_start(arginput):
+    import os
+# Take binary file back up to '_orig.bin'
+    filesize = os.path.getsize(arginput + ".bin")
+    if filesize > 0x4F000:
+# Remove gap between Flash A and B 
```

Comment:
Why is the gap between bank A and B removed? Does the flash algo used for the ncs36510 translate these addresses accordingly?